ldapom/error.py

```diff
@@ -29,3 +29,6 @@ class LDAPAttributeNameNotFoundError(LDAPomError):
 
 class LDAPCouldNotFetchAttributeTypes(LDAPomError):
     pass
+
+class LDAPomReadOnlyError(LDAPomError):
```

Comment:
As only the base error uses `LDAPom`, I would skip the `om` in the class name here.